cmd/lncli/commands.go

```diff
@@ -2566,12 +2566,17 @@ var addInvoiceCommand = cli.Command{
 				"specified an expiry of 3600 seconds (1 hour) " +
 				"is implied.",
 		},
-		cli.BoolTFlag{
+		cli.BoolFlag{
 			Name: "private",
 			Usage: "encode routing hints in the invoice with " +
 				"private channels in order to assist the " +
 				"payer in reaching you",
 		},
+		cli.StringFlag{
+			Name: "private_hops",
+			Usage: "a json array string of private channel ids " +
+				"to use as hop hints",
```

Comment:
Why not the simpler format `--private_hops 100,200,300`?

Reply:
Done